astropy/io/fits/column.py

```diff
@@ -461,7 +461,9 @@ def __init__(self, name=None, format=None, unit=None, null=None,
                  array=None, ascii=None):
         """
         Construct a `Column` by specifying attributes.  All attributes
-        except `format` can be optional.
+        except format can be optional, see :ref:`column_creation` and
```

Comment:
Here, instead of just "format", changes this to:
```
``format``
```
with double-backticks.  This indicates that it represents some variable or argument, but by using double backticks instead of single-backticks it won't try to turn it into an API link.